### PR TITLE
fix two links in the rst files

### PR DESCRIPTION
**Pull request by razr**

### CODE REVIEW

**Review by LanderU — APPROVED** [review]
LGTM!

File: source/index.rst [hunk 1]
```diff
@@ -18,5 +18,6 @@ The Real-Time Working Group's mission is to advocate for and work on memory mana
    Subprojects/subprojects
    Related-Projects/related_projects
    Resources/resources
+   rtos/index
```
Comment (carlossvg) [inline source/index.rst]: @razr I think I added this by mistake. Could you remove the rtos folder?

Comment (razr) [reply source/index.rst]: done

**Review by carlossvg — APPROVED** [review]
LGTM

**Pull request merged**